### PR TITLE
Set testnet and mainnet heights for Lima hardfork (aiming 16th and 30th of Oct)

### PR DESCRIPTION
**Pull request by sennui**

### CODE REVIEW

File: docs/release-notes/next-lima/PT-168425449-lima-heights.md [hunk 1]
```diff
@@ -0,0 +1 @@
+* Set Lima testnet hardfork height to 154400 (16th Oct 2019, 2pm CEST) and Lima mainnet to 161120 (30th Oct 2019, 2pm CEST)
```
Comment (dincho) [inline docs/release-notes/next-lima/PT-168425449-lima-heights.md]: maybe split on two lines for better readability

Comment (sennui) [reply docs/release-notes/next-lima/PT-168425449-lima-heights.md]: Will wait for the airdrop development and fix with final version

**Comment by sennui** [comment]
@dincho PoW is probabilistic ;) there is no zero chance that my comments are exact ;)
(had to beautify the numbers)

**Comment by sennui** [comment]
Obviously the numbers are no longer accurate as we move the release

File: docs/release-notes/next-lima/PT-168425449-lima-heights.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Set Lima testnet hardfork height to 154460 (16th Oct 2019, 2pm CEST)
+  and Lima mainnet to 161120 (30th Oct 2019, 2pm CEST)
```
Comment (ThomasArts) [inline docs/release-notes/next-lima/PT-168425449-lima-heights.md]: This should be CET, since summer time has ended by then!

**Pull request merged**